devices/lonsonho.js

```diff
@@ -32,6 +32,26 @@ module.exports = [
             exposes.enum('backlight_mode', ea.ALL, ['LOW', 'MEDIUM', 'HIGH']),
             exposes.numeric('calibration_time', ea.STATE).withUnit('S').withDescription('Calibration time')],
     },
+    {
+        fingerprint: [{modelID: 'TS130F', manufacturerName: '_TZ3000_j1xl73iw'}],
+        model: 'TS130F_dual',
```

Comment:
Is there any model number written on the device? If yes please use that, and rename the picture/md for the docs to this.

Reply:
Unfortunately no, I've checked it now.. all the other sides of the device are empty.. manual also doesn't specify any model number.. It's called '2CH Zigbee curtain module' even in manual.. App in manual is Tuya, I'm guessing it's Lonsonho only by seller mentioning that info on [aliexpress](https://www.aliexpress.com/item/1005003044227983.html)..
![image](https://user-images.githubusercontent.com/7549001/151213485-e23877b8-6c0a-4c24-a736-0a449e7ec8db.png)
manual:
![image](https://user-images.githubusercontent.com/7549001/151214844-8f1b0e2b-626a-472f-b194-e83cc239330b.png)

Reply:
ok thanks